packages/nimble-components/src/table/specs/table-data-api.md

```diff
@@ -0,0 +1,130 @@
+# Table Data API
```

Comment:
You can't do this until #828 completes, but it would be good to link to this document from that one.